### ISSUE TITLE
Filter dropdown swaps back to active on every page load

### ISSUE THREAD

**Issue by apolopena** [gitpod-io/gitpod#3875]
### Bug description
**In the New workspace dashboard**:
Two items to the left of the workspace button is the filter dropdown. I understand why the filter drowdown defaults to 'active' and that is fine however it is too agressive to switch that status back to 'active' on every page load. Perhaps a 1 hour cookie that keep this dropdown on the choice the user last left it is in order?

The perfect example of this being a bug-like hinderance or leaving the user feeling that their descisions have been 'hijacked', is in the case of them deleting a series of non active workspaces. This flow is described below in the steps to reproduce section

### Steps to reproduce

In your workspace dashboard with multiple workspaces active and not active. Go about deleting some of your unused workspaces:
1. Change dropdown filter to 'All'
2. Click a workspaces hamburger
3. Click Delete
4. Acknowledge delete
5. Page reloads
6. The filter dropdown is reset to 'Active'
7. The workspaces the user was trying to delete have disapeared
8. The user now how to repeat 5 steps to delete a single workspace.

### Expected behavior

A user should have the option to make their choices in the UI 'stick', even if this is for a limited time. 

### Example repository

Any

**Comment by gtsiolis** [comment]
Hey @apolopena! I cannot reproduce step 4. Page does not reload and the filter remains on _All_. Is this related to https://github.com/gitpod-io/gitpod/issues/3876?

**Comment by apolopena** [comment]
@gtsiolis 
No this issue is unrelated to  #3876.
I will go over my repro steps again and make sure I wrote them down properly.

**Comment by apolopena** [comment]
@gstiolis
After going through my reproduction steps again it turns out this issue does indeed appear to be releated to https://github.com/gitpod-io/gitpod/issues/3876. This is odd because I did not get the same results the second time I reproduced this issue. The only difference in the reproductions steps I took might be that the first time around I was deleting allready timed out workspaces and the second time around I was deleting workspaces that I had stopped manually.

This second time around, all stopped workspaces were deleted all at once and the page reloaded but the filter dropdown stayed on All.
